
Although now removed in gnulib:
	https://github.com/coreutils/gnulib/commit/329877ac214ca86f7adb6a59f
coreutils has not yet updated.

diff -wpruN --no-dereference '--exclude=*.orig' a~/lib/localename-unsafe.c a/lib/localename-unsafe.c
--- a~/lib/localename-unsafe.c	1970-01-01 00:00:00
+++ a/lib/localename-unsafe.c	1970-01-01 00:00:00
@@ -48,7 +48,7 @@
 # if defined __sun
 #  if HAVE_GETLOCALENAME_L
 /* Solaris >= 12.  */
-extern char * getlocalename_l(int, locale_t);
+extern const char * getlocalename_l(int, locale_t);
 #  elif HAVE_SOLARIS114_LOCALES
 #   include <sys/localedef.h>
 #  endif
